index.js

```diff
@@ -118,6 +118,9 @@ const gulpSass = (options, sync) => through.obj((file, enc, cb) => { // eslint-d
     return cb(new PluginError(PLUGIN_NAME, error));
   };
 
+  // eslint-disable-next-line global-require, import/no-extraneous-dependencies
+  const compiler = gulpSass.compiler || require('node-sass');
```

Comment:
Should not this be auto-detection?
```js
let compiler = gulpSass.compiler;
if (!compiler) {
  try {
    compiler = require('node-sass');
  } catch (e) {
    try {
      compiler = require('sass');
    } catch (e) {
      return cb(new PluginError(
        PLUGIN_NAME,
        "No compiler found!\nPlease install one of the npm packages: node-sass or sass, or set compiler in code:\n    const sass = require('gulp-sass');\n    sass.compiler = require('sass');\n"
      ));
    }
  }
}
```

So that user don't need to set dart-sass by hand. For reference, webpack's sass-loader does this kind of auto-detection.

Reply:
That makes a lot of sense, however I'm not in a position to fix this at the moment.
I'll probably get to it somewhere next week (if nobody else has yet).